articles/services/index.md

```diff
@@ -0,0 +1,42 @@
+---
+url: /services
+classes: topic-page
+title: Auth0 Professional Services
+description: Professional Services offered by Auth0.
+---
+
+<div class="topic-page-header">
+  <div data-name="example" class="topic-page-badge"></div>
+  <h1>Auth0 Professional Services</h1>
+  <p>
+    The Auth0 Professional Services Portfolio consists of innovative, specialized service offerings designed to optimize customers investments on the Auth0 Platform. These offerings help customers deploy, implement and maintain solutions based on proven practices, avoiding common pitfalls and reducing risk.
+  </p>
+</div>
+
+## Assess & Plan
+
+<%= include('../_includes/_topic-links', { links: [
+  'services/architectural-design',
+  'services/performance-scalability',
+  'services/geo-ha',
+  'services/scenario-guidance',
+  'services/code-review'
+] }) %>
+
+## Maintain & Improve
+
+<%= include('../_includes/_topic-links', { links: [
+  'services/pair-programming',
+  'services/private-saas-configuration',
+  'services/code-review',
```

Comment:
@eugeniop code-review is listed twice, under **Assess & Plan** and **Maintain & Improve**. Just double-checking that this was not done by mistake.